src/domains/fdr-app/api/fdr-fase1/nodoPerPa/v1/fdr_pagopa_nodoChiediElencoFlussiRendicontazione.xml.tpl

```diff
@@ -11,28 +13,30 @@
+                  var dom2parse = ((string) context.Variables["readrequest"]);
 
-                String[] tagBrokerId = {"identificativoIntermediarioPA"};
-                String[] result = dom2parse.Split(tagBrokerId, StringSplitOptions.RemoveEmptyEntries);
-                String brokerId = result.Length > 2 ? result[1].Substring(1,result[1].Length-3).Replace("xmlns=\"\">", "") : "ND";
+                  String[] tagBrokerId = {"identificativoIntermediarioPA"};
+                  String[] result = dom2parse.Split(tagBrokerId, StringSplitOptions.RemoveEmptyEntries);
+                  String brokerId = result.Length > 2 ? result[1].Substring(1,result[1].Length-3).Replace("xmlns=\"\">", "") : "ND";
 
-                String[] tagStationId = {"identificativoStazioneIntermediarioPA"};
-                result = dom2parse.Split(tagStationId, StringSplitOptions.RemoveEmptyEntries);
-                String stationId = result.Length > 2 ? result[1].Substring(1,result[1].Length-3).Replace("xmlns=\"\">", "") : "ND";
+                  String[] tagStationId = {"identificativoStazioneIntermediarioPA"};
+                  result = dom2parse.Split(tagStationId, StringSplitOptions.RemoveEmptyEntries);
+                  String stationId = result.Length > 2 ? result[1].Substring(1,result[1].Length-3).Replace("xmlns=\"\">", "") : "ND";
 
-                String[] tagDomainId = {"identificativoDominio"};
-                result = dom2parse.Split(tagDomainId, StringSplitOptions.RemoveEmptyEntries);
-                String domainId = result.Length > 2 ? result[1].Substring(1,result[1].Length-3).Replace("xmlns=\"\">", "") : "ND";
+                  String[] tagDomainId = {"identificativoDominio"};
+                  result = dom2parse.Split(tagDomainId, StringSplitOptions.RemoveEmptyEntries);
+                  String domainId = result.Length > 2 ? result[1].Substring(1,result[1].Length-3).Replace("xmlns=\"\">", "") : "ND";
 
-                String[] tagPspId = {"identificativoPSP"};
-                result = dom2parse.Split(tagPspId, StringSplitOptions.RemoveEmptyEntries);
-                String pspId = result.Length > 2 ? result[1].Substring(1,result[1].Length-3).Replace("xmlns=\"\">", "") : "ND";
+                  String[] tagPspId = {"identificativoPSP"};
+                  result = dom2parse.Split(tagPspId, StringSplitOptions.RemoveEmptyEntries);
+                  String pspId = result.Length > 2 ? result[1].Substring(1,result[1].Length-3).Replace("xmlns=\"\">", "") : "ND";
 
-                return "fdr::fase1::cachereq::" + brokerId + "-" + stationId + "-" + domainId + "-" + pspId;
-              }" />
+                  return "fdr::fase1::cachereq::" + brokerId + "-" + stationId + "-" + domainId + "-" + pspId;
+                }" />
+              </when>
               <cache-lookup-value variable-name="fdr_cached_response_uuid" key="@((string) context.Variables["cached_response_key"])" caching-type="internal" default-value="NONE"/>
```

Comment:
cache-lookup-value should be done iif `enable_fdr_nodoChiediElenco_cache_flag` its true